layers/meta-resin-allwinner/conf/machine/bananapi-m1-plus.conf

```diff
@@ -0,0 +1,9 @@
+#@TYPE: Machine
+#@NAME: Bananapi M1+
+#@DESCRIPTION: Machine configuration for the Bananapi M1+, based on allwinner A20 CPU http://bananapi.org/
+
+require conf/machine/include/sun7i.inc
+
+KERNEL_DEVICETREE = "sun7i-a20-bananapi-m1-plus.dtb"
+UBOOT_MACHINE = "bananapi_m1_plus_config"
+SUNXI_FEX_FILE = "sys_config/a20/Bananapi.fex"
```

Comment:
Please change this machine conf to base on the existing bananapi.conf machine definition.
Use https://github.com/resin-os/resin-jetson-tx2/blob/master/layers/meta-resin-jetson/conf/machine/skx2.conf as reference